lib/src/modern_player_options.dart

```diff
@@ -314,6 +314,31 @@ class ModernPlayerCustomActionButton {
       {required this.icon, this.onPressed, this.onDoubleTap, this.onLongPress});
 }
 
+/// The top level type which defines the strategies for selecting tracks
+sealed class DefaultSelector {}
+
+/// Will default to disabling this track
+class DefaultSelectorOff extends DefaultSelector {}
+
+/// Provide custom logic for selecting a track
+/// It will pick the first track that returns true
+class DefaultSelectorCustom extends DefaultSelector {
```

Comment:
If there is no need of DeaultSelectorOff, then please rename it to "DeafultSelector" and there is also no need of sealed class then.